src/Emulator/Peripherals/Peripherals/DMA/ATCDMAC100.cs

```diff
@@ -0,0 +1,578 @@
+//
+// Copyright (c) 2010-2023 Antmicro
```

Comment:
I guess it should be some RapidSilicon license rather that this one form Antmicro. We can check it from other code in EmbeddedCommon-dev